aiidalab_widgets_base/viewers.py

```diff
@@ -168,44 +174,40 @@ class _StructureDataBaseViewer(ipw.VBox):
     DEFAULT_SELECTION_RADIUS = 6
     DEFAULT_SELECTION_COLOR = "green"
 
-    def __init__(self, configure_view=True, **kwargs):
+    def __init__(
+        self,
+        configure_view=True,
+        configuration_tabs=["Selection", "Appearance", "Cell", "Download"],
+        default_camera="orthographic",
+        **kwargs,
+    ):
         # Defining viewer box.
 
-        # 1. Nglviwer
+        # Nglviwer
         self._viewer = nglview.NGLWidget()
-        self._viewer.camera = "orthographic"
+        self._viewer.camera = default_camera
         self._viewer.observe(self._on_atom_click, names="picked")
         self._viewer.stage.set_parameters(mouse_preset="pymol")
 
-        # 2. Camera type.
-        camera_type = ipw.ToggleButtons(
-            options={"Orthographic": "orthographic", "Perspective": "perspective"},
-            description="Camera type:",
-            value="orthographic",
-            layout={"align_self": "flex-end"},
-            style={"button_width": "115.5px"},
-            orientation="vertical",
-        )
-
-        def change_camera(change):
-
-            self._viewer.camera = change["new"]
-
-        camera_type.observe(change_camera, names="value")
-        view_box = ipw.VBox([self._viewer, camera_type])
+        view_box = ipw.VBox([self._viewer])
 
         # Constructing configuration box
-        if configure_view:
+        if configure_view and len(configuration_tabs) != 0:
```

Comment:
Maybe a deprecation warning would be a good addition, something like this:

```suggestion
        if configure_view is not None:
            warnings.warn("`configure_view` is deprecated, please use `configuration_tabs` instead.", DeprecationWarning)
        if configure_view and len(configuration_tabs) != 0:
```

Reply:
I am fine with deprecation warning, but the if condition is tricky because right now `configure_view=True` by default. 
If we make it None by default, then the if condition `if configure_view and len(configuration_tabs)` does not work.

Reply:
I guess keeping the default to True and just inverting the condition like this will work, though it is a bit confusing:
```python
if not configure_view:
    warnings.warn("`configure_view` is deprecated, please use `configuration_tabs` instead.", DeprecationWarning)
```

Reply:
@csadorf thanks, I went with your idea.